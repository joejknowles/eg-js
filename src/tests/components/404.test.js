import Component from '../../components/404';

import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

const shallowRenderer = ReactTestUtils.createRenderer();

it('renders 404 page as before', () => {
  const component = shallowRenderer.render(<Component />);
  expect(component).toMatchSnapshot();
});
