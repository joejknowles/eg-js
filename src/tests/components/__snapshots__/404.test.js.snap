exports[`test renders 404 page as before 1`] = `
<h1>
  404 - page not found. Oops
</h1>
`;
